### PR TITLE
Fixed process encodings

### PR DESCRIPTION
**Pull request by Innos**
Closes SoftUni-Internal/suls-issues/issues/2206 - Possible encoding problem causes different results with the same code #2206

### CODE REVIEW

File: Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs [hunk 1]
```diff
@@ -24,12 +31,6 @@ public enum ExecutionStrategyType
...
         MySqlPrepareDatabaseAndRunQueries = 16,
         MySqlRunQueriesAndCheckDatabase = 17,
-        MySqlRunSkeletonRunQueriesAndCheckDatabase = 18,
-        CSharpUnitTestsExecutionStrategy = 23,
-        CSharpProjectTestsExecutionStrategy = 24,
-        JavaProjectTestsExecutionStrategy = 25,
-        CPlusPlusZipFileExecutionStrategy = 26,
-        JavaUnitTestsExecutionStrategy = 27,
-        CSharpAspProjectTestsExecutionStrategy = 28
+        MySqlRunSkeletonRunQueriesAndCheckDatabase = 18
```
Comment (jicata) [inline Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs]: Reason behind reordering?

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/ExecutionStrategy.cs [hunk 2]
```diff
@@ -29,11 +33,26 @@ protected ExecutionResult CompileExecuteAndCheck(ExecutionContext executionConte
...
+                    null,
+                    false,
+                    useSystemEncoding);
+                var testResult = this.ExecuteAndCheckTest(
+                    test,
+                    processExecutionResult,
+                    checker,
+                    processExecutionResult.ReceivedOutput);
                 result.TestResults.Add(testResult);
             }
```
Comment (jicata) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/ExecutionStrategy.cs]: Blank lines

File: Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs (hunk 1 above)
Comment (jicata) [inline Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs]: Reason behind reordering?

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/ExecutionStrategy.cs (hunk 2 above)
Comment (jicata) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/ExecutionStrategy.cs]: Blank lines

File: Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs (hunk 1 above)
Comment (Innos) [inline Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs]: Strategies are now ordered by language in groups (makes it more readable and easier to find what you're looking for). I looked online and found out that such orderings are common as they bring semantic benefits.

**Pull request merged**